### PR TITLE
issue #34 - fix/size buttons md 

### LINKED ISSUE #34: Right And Left Slider Button Is Very Small On Large Screens (1044X827)
### Explanation.
At a resolution of 1044x827 the button sizes are very small and therefore hardly visible.

### CHECKLIST 

- [ ] Adjust the buttons size
- [ ] Make the buttons size more visible in a resolution of 1044x827 or higher.
### References.
https://www.figma.com/file/zzEwuPfGTiZh4EA11MyZ5c/Sencico?node-id=105%3A83

### LINKED ISSUE #43: Fix slider arrow position in small screens
### Explanation
On small screens the slider arrows look unaligned (check screenshots), this slider use the library React Slick, you can check the documentation here: https://react-slick.neostack.com/ . This library allow the use padding and other CSS properties over their parts. To fix this problem please check if something is affecting the arrows and check how you can fix it using CSS and the documentation.

### Screenshots
imagen

### Checklist
 This issue can be closed when the following tasks are completed:
 
 - [ ] The arrows are centered in small screens.
 
### References
- https://react-slick.neostack.com/

### PR DESCRIPTION
**Pull request by Eghinner**
Issue #34 
In width 827px
screenshot-localhost_3000-2022 02 22-14_40_00
Changes in Slider component are to replace Arrows components for backgroundImage
In summary less code

### CODE REVIEW

**Review by heliomar-pena — APPROVED** [review]
I see good the code changes and the visual styles, good job!

@angelik0828 please check and confirm visual styles

Before:
imagen

Now:
imagen

**Review by Israel-Laguan — APPROVED** [review]
I like SVG more ;<

File: src/components/Slider.js [hunk 1]
```diff
@@ -47,12 +28,13 @@ const Next = (props) => {
       className={`${className} arrow-custom-circle`}
       style={{
         ...style,
-        color: pathname !== '/' ? "#FFF" : "#11698E",
-        background: pathname !=='/' ? "no-repeat center #11698E" : "no-repeat center #FFF"
+        background:
+        pathname === '/'
+        ? "center no-repeat #FFF url('https://img.icons8.com/material-rounded/24/11698E/chevron-right.png')"
+        : "center no-repeat #11698E url('https://img.icons8.com/material-rounded/24/FFFFFF/chevron-right.png')"
```
Comment (heliomar-pena) [inline src/components/Slider.js]: Please change #FFF for `#F8F1F1`

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
A change that I notice

File: src/components/Slider.js (hunk 1 above)
Comment (Eghinner) [reply src/components/Slider.js]: change made.

File: src/components/Slider.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const Prev = (props) => {
         ...style,
         background:
         pathname === '/'
-        ? "center no-repeat #FFF url('https://img.icons8.com/material-rounded/24/11698E/chevron-left.png')"
+        ? "center no-repeat #F8F1F1 url('https://img.icons8.com/material-rounded/24/11698E/chevron-left.png')"
         : "center no-repeat #11698E url('https://img.icons8.com/material-rounded/24/FFFFFF/chevron-left.png')"
```
Comment (heliomar-pena) [inline src/components/Slider.js]:
```suggestion
        : "center no-repeat #11698E url('https://img.icons8.com/material-rounded/24/F8F1F1/chevron-left.png')"
```

File: src/components/Slider.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ const Next = (props) => {
         ...style,
         background:
         pathname === '/'
-        ? "center no-repeat #FFF url('https://img.icons8.com/material-rounded/24/11698E/chevron-right.png')"
+        ? "center no-repeat #F8F1F1 url('https://img.icons8.com/material-rounded/24/11698E/chevron-right.png')"
         : "center no-repeat #11698E url('https://img.icons8.com/material-rounded/24/FFFFFF/chevron-right.png')"
```
Comment (heliomar-pena) [inline src/components/Slider.js]:
```suggestion
        : "center no-repeat #11698E url('https://img.icons8.com/material-rounded/24/F8F1F1/chevron-right.png')"
```

File: src/components/Slider.js (hunk 1 above)
Comment (heliomar-pena) [reply src/components/Slider.js]: Thanks!

**Comment by angelik0828** [comment]
In the view of the categories in the slider, the buttons are very close together, there should be a margin that separates the buttons from the arrows in Resolution 1044, another thing is that the buttons of the slider are a bit small for the size of the button, image attached guide. @Eghinner 
The color of the pressed button is ´#F8F1F1´
image

**Comment by Eghinner** [comment]
New changes:

- Size of the arrows buttons
- Change active slider button backgroundColor
- Add margin to slick-slider in resolution 1280px or lower

- screenshot-localhost_3000-2022 02 25-11_47_19

- screenshot-localhost_3000-2022 02 25-11_48_02

**Review by heliomar-pena — APPROVED** [review]
Thanks for the change! Very good work!

**Review by angelik0828 — APPROVED** [review]

**Comment by angelik0828** [comment]
I see the change looks good good job @Eghinner

**Pull request merged**